src/helpers/firebase.ts

```diff
@@ -13,4 +14,4 @@ const firebaseConfig = {
 
 const app = initializeApp(firebaseConfig);
 
-export const auth = getAuth(app);
+export const auth = tryConst(() => getAuth(app));
```

Comment:
I personally dislike the way it called (cant prove my point tho), but maybe change it to `tryConst(getAuth, app);`?